ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -282,7 +282,7 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
         }
 
         if customAccessoryView == nil {
-            accessoryView = accessoryButtonTitle != "" ? createAccessoryButton(withTitle: accessoryButtonTitle) : nil
+            accessoryButton = accessoryButtonTitle != "" ? createAccessoryButton(withTitle: accessoryButtonTitle) : nil
```

Comment:
I am not sure why we decided to have customaccessoryView and accessoryView then? can we just use one?